HPXMLtoOpenStudio/resources/vehicle.rb

```diff
@@ -0,0 +1,223 @@
+# frozen_string_literal: true
+
+# Collection of methods for adding vehicle-related OpenStudio objects, built on the Battery class
+class Vehicle
+  # Adds any HPXML Vehicles to the OpenStudio model.
+  # Currently only models electric vehicles.
+  #
+  # @param runner [OpenStudio::Measure::OSRunner] Object typically used to display warnings
+  # @param model [OpenStudio::Model::Model] OpenStudio Model object
+  # @param spaces [Hash] Map of HPXML locations => OpenStudio Space objects
+  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
+  # @param schedules_file [SchedulesFile] SchedulesFile wrapper class instance of detailed schedule files
+  # @return [nil]
+  def self.apply(runner, model, spaces, hpxml_bldg, schedules_file)
+    hpxml_bldg.vehicles.each do |vehicle|
+      next unless vehicle.vehicle_type == HPXML::VehicleTypeBEV
```

Comment:
We should also add a warning for non-BEV vehicles to make it clear we are not modeling them. Something like [this](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/misc_loads.rb#L20-L34). And add a test in test_validation.rb.

Reply:
Addressed in https://github.com/NREL/OpenStudio-HPXML/pull/1533/commits/97450dca0c343296c903278bacdf310b21ef6699